examples/examples_nodejs_test.go

```diff
@@ -29,6 +35,46 @@ func TestGetTs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestCustomResourceEmulator(t *testing.T) {
+	crossTest := func(t *testing.T, outputs auto.OutputMap) {
+		require.Contains(t, outputs, "cloudformationAmiId")
+		cloudformationAmiId := outputs["cloudformationAmiId"].Value.(string)
+		require.NotEmpty(t, cloudformationAmiId)
+
+		require.Contains(t, outputs, "emulatorAmiId")
+		emulatorAmiId := outputs["emulatorAmiId"].Value.(string)
+		assert.Equal(t, cloudformationAmiId, emulatorAmiId)
+	}
+
+	cwd := getCwd(t)
+	options := []opttest.Option{
+		opttest.LocalProviderPath("aws-native", filepath.Join(cwd, "..", "bin")),
+		opttest.YarnLink("@pulumi/aws-native"),
+	}
+	test := pulumitest.NewPulumiTest(t, filepath.Join(cwd, "cfn-custom-resource"), options...)
+	test.SetConfig(t, "amiRegion", "us-west-2")
+
+	previewResult := test.Preview(t)
```

Comment:
`Check` will receive unknowns during preview. (even if supports preview is not enabled).
I'll change `Check` to handle this, it should be relatively easy

Reply:
Done